DbService/src/epicsTool_main.cc

```diff
@@ -0,0 +1,105 @@
+//
+// main to drive a command line interface to EpicsTool,
+// which reads data from the EPICS archive database
+//
+
+#include "Offline/DbService/inc/EpicsTool.hh"
+#include <iostream>
+#include <string>
+#include <vector>
+
+int main(int argc, char** argv) {
+  std::vector<std::string> words;
+
+  bool doHelp = false;
+  bool doNames = false;
+  std::string word, var, time;
+  size_t i = 1;
+  size_t lim(argc);
+  while (i < lim) {
```

Comment:
Please consider using boost::program_options here for more maintainable code.

Reply:
@kutschke @brownd1978 do we have a policy?  I've gone around and around on this (and other similar things) - if we declare a standard, I'll just follow it.

Reply:
I don't think that we have ever discussed a policy - we have so few main programs.   I have also written my own command line parser in at least one of the mains. I think it would be good to move to a standard command line parser too as targets of opportunity.  

Are there reasonably options other than boost::program_options  .  I will ask Kyle if anything is in the standard, now or upcoming.

Reply:
Thank you for updating  the code to use boost::program_options